libs/hdf-converters/src/ckl-mapper/checklist-jsonix-converter.ts

```diff
@@ -650,6 +698,31 @@ export class ChecklistJsonixConverter extends JsonixIntermediateConverter<
       : '';
   }
 
+  // computes what the impact would be based on the given tags
+  computeImpact(
```

Comment:
It turns out that this functionality (computing impact from severity) is not a inspecJs thing. It is, however, done a lot within the mappers through the base converter's `impactMapping` function. In this case, it also doesn't matter what critical maps to (.9 was chosen because it seems to be most common), because this value is only ever used to convert from ckl `thirdPartyTools.hdfSpecificData.severity` to `control.impact`, but if that mapping is wrong, there would be a `thirdPartyTools.hdfSpecificData.impact` to correct for it. The appropriate changes were made.